docs/buildspecs/buildspec_overview.rst

```diff
@@ -447,6 +448,28 @@ Let's build the test and see the output.
 
    .. command-output:: buildtest build -b tutorials/test_status/file_and_dir_check.yml
 
+.. _symlink_check:
+
+Passing test based on Symbolic Link
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+buildtest can configure PASS/FAIL of test based on the status of symbolic link. This can be useful if your test will create a symbolic link to a file or directory and 
+will pass if the symbolic link is not broken.  
```

Comment:
could replace `and will pass if the symbolic link is not broken` -> `and test will pass if the symbolic link is present.`